dags/collect_weekly.py

```diff
@@ -33,6 +34,7 @@
     # 'sla_miss_callback': yet_another_function,
     # 'trigger_rule': 'all_success'
 }
+content_sources = os.getenv("CONTENT_SOURCES", "Twitter,Reddit,Article,Youtube,RSS")
```

Comment:
:+1: Thank you for the PR for exposing the options; that's valuable for users to customize the sources.

I'm outside, but I will test it locally later; from the high level, this may not work as expected due to the dags being managed by airflow. During the execution in the runtime, this path doesn't have a chance to pick up those environment variables. Could you help to double-check on it? Maybe we could also consider controlling the sources in `af_xxx.py` entries.

Reply:
Sure!

Reply:
@finaldie So, yeah, you were right, inside dags it isn't working idk why. 

Also, we can add some sort of the API keys checking, so if they are empty, just ignore sorces